backend/infrahub/git/integrator.py

```diff
@@ -528,6 +530,22 @@ async def import_schema_files(self, branch_name: str, commit: str, config_file:
                 f"schema '{schema_file.identifier}' loaded successfully!", repository=self.name, commit=commit
             )
 
+        max_wait_time = 60
```

Comment:
Should we have these kinds of values in the settings, maybe in an section named "advanced"?

Reply:
Any idea with regards to what to call such a setting? You mean that we'd create a new class of settings called `AdvancedSettings`?

Reply:
Exactly, that's the hard part, naming things, maybe something like `AdvancedSettings.schema_convergence_max_wait_time`.

Reply:
based on the other comment, in my opinion this setting should be in the SDK